setup.py

```diff
@@ -33,5 +33,8 @@
         "Programming Language :: Python :: 3.13",
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
```

Comment:
you could also add the `Typing :: Typed` PyPI classifier now

Reply:
FYI I added it in https://github.com/data-apis/array-api-compat/pull/255